contracts/INKPurchase.sol

```diff
@@ -9,22 +9,24 @@ contract INKPurchase is Ownable {
 	using SafeERC20 for IERC20;
 
 	address public immutable usdc;
+	address public treasuryWallet;
 	address public constant weth9 =
-		0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+		0xc778417E063141139Fce010982780140Aa0cD5Ab;
```

Comment:
This address must be set in constructor when deploying because this code should be reused for both ETH and Polygon, and on those chains, WETH addresses can be different.